plugins/mobile_channel_header_plug/mobile_channel_header_plug.jsx

```diff
@@ -28,14 +28,16 @@ export default class MobileChannelHeaderPlug extends React.PureComponent {
 
     createButton(plug) {
         return (
-            <button
-                className='navbar-toggle navbar-right__icon'
-                onClick={() => this.fireAction(plug)}
-            >
-                <span className='icon navbar-plugin-button'>
-                    {plug.icon}
-                </span>
-            </button>
+            <li className='flex-parent--center'>
```

Comment:
nit: should we be using li when we don't have a ul/ol?

Reply:
The <ul> is there on the parent for hte whole dropdown menu.